lib/pacio_test_kit/interactions_test.rb

```diff
@@ -53,6 +86,18 @@ def validate_resource_id(resource, id, req_num)
       add_message('error', bad_resource_id_message(id, req_num))
     end
 
+    def validate_resource_creation_time(create_request, req_num)
+      return if create_request.response[:created_at].present?
+
+      add_message('error', no_time_created_message(create_request, req_num))
+    end
+
+    def validate_resource_body(create_request_body, create_response_body, req_num)
+      return if create_request_body == create_response_body
```

Comment:
This statement will always be false because the id and metadata field will be different. We will not have to check the content anyway, as having the same content is a `SHOULD` requirement, not `SHALL`